### PR TITLE
Add clean build folder fixes #11

### LINKED ISSUE #11: Add clean build directory
### Description
Before build starts should clean the build directory.  This way artifacts from previous builds will be removed.

#### Example
Could add to https://github.com/Polymer/polymer-build/blob/master/test/test-project/gulpfile.js#L28

```
const del = require('del');

// Clean build directory
gulp.task('clean', () => {
  return del(['build/**','!build']);
});

gulp.task('default',['clean'], (cb) => {
   ...
```

### PR DESCRIPTION
**Pull request by chuckh**
Before build starts should clean the build directory. This way artifacts from previous builds will be removed. Fixes #11.

### CODE REVIEW

File: test/test-project/gulpfile.js [hunk 1]
```diff
@@ -26,7 +27,12 @@ let project = new PolymerProject({
   shell: 'shell.html',
 });
 
-gulp.task('test1', () => {
+// Clean build directory
+gulp.task('clean', () => {
+  return del('build');
+});
+
+gulp.task('test1',['clean'], (cb) => {
```
Comment (FredKSchott) [inline test/test-project/gulpfile.js]: nit: `,['clean'` -> `, ['clean'`

File: test/test-project/gulpfile.js [hunk 2]
```diff
@@ -1,3 +1,4 @@
+const del = require('del');
```
Comment (FredKSchott) [inline test/test-project/gulpfile.js]: del isn't a dependency of this project, can you add it as a devDependency?

Comment (chuckh) [inline test/test-project/gulpfile.js]: Fixed, del has been added.

Comment (chuckh) [inline test/test-project/gulpfile.js]: Fixed, added space.

**Comment by FredKSchott** [comment]
nice! Thanks for the PR!

**Pull request merged**